docs/medical-api/api-reference/fhir/get-medical-record-summary.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: "Get Medical Record Summary"
+description: "Get the Medical Record Summary in PDF or HTML formats."
+api: "GET /patient/:id/medical-record"
+---
+
+This endpoint returns the URL to download the patient's Medical Record Summary in the PDF or HTML format, if it exists. 
+
+To create a Medical Record Summary, you need to use the [Start Consolidated Data Query](/medical-api/api-reference/fhir/consolidated-data-query-post). 
+
+## Path Params
+
+<ParamField path="id" type="string" required>
```

Comment:
nit: its patientId in the `getMedicalRecordSummary` function, so just make them consistent

Reply:
It looks like we use this kinda structure everywhere - the route just uses `/patient/:id/` and the SDK uses anyFunction(patientId: string) {}. I'll skip this if thats ok